docs/target.settings.yaml

```diff
@@ -4,7 +4,18 @@ general:
   cm_device_codename: grouper
   cm_release_type: snapshot
   cm_release_version: M12
-  parent_template: mia-default
+  parent_templates:
+    - mia-default
+    - wifi-only
```

Comment:
Ah interesting :) I'm just on mobile, but lemme think through this later? wondering if there are implications of multiple parents in one template, so I'd only been considering one. So you thinking later templates in array take precedence when merging yaml and directory structures?